frontend/src/scenes/events/UsageDisabledWarning.tsx

```diff
@@ -4,8 +4,7 @@ import React from 'react'
 export function UsageDisabledWarning({ tab }: { tab: string }): JSX.Element {
     return (
         <Alert
-            type="info"
-            showIcon
+            style={{ marginBottom: 16, display: 'flex', alignItems: 'center' }}
```

Comment:
styles!! 👿 

Could we use classNames here instead? If not, could we quickly refactor this over to the [AlertMessage](https://storybook.posthog.net/?path=/docs/lemon-ui-alert-message--info) component which does support it?

Reply:
ah good callout!